src/Features/Core/Portable/ExternalAccess/VSTypeScript/VSTypeScriptWellKnownDiagnosticCustomTags.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.VSTypeScript
+{
+    internal static class VSTypeScriptWellKnownDiagnosticCustomTags
```

Comment:
I never remember the rules correctly, but introducing this new class means that we (TypeScript VS) will also need to bump our Roslyn version to access this tag? @mavasani